Graphics/ShaderTools/CMakeLists.txt

```diff
@@ -2,6 +2,8 @@ cmake_minimum_required (VERSION 3.6)
 
 project(Diligent-ShaderTools CXX)
 
+set(DILIGENT_NO_DXC FALSE CACHE INTERNAL "Use DirectXShaderCompiler compiler")
```

Comment:
I don't think this needs to be a cache variable. If it is internal and not an CMake option, there is no need to define this variable here at all.

Reply:
Besides, this variable defines whether to **disable** using the compiler, not when to use it

Reply:
Please remove this variable, it is not needed